src/state.cuh

```diff
@@ -2406,6 +2451,8 @@ public:
     )
     {
         bn_t gas_cost, set_gas_refund;
+
+        // todo missing check for eip-1706?
```

Comment:
I check for it in the operation. eip-2200 is saying to verify the gas left before computing the gas cost/refund for sstore operation, which a little different than the initial eip-1706.